resources/[system]/scoreboard/server.lua

```diff
@@ -0,0 +1,101 @@
+RegisterNetEvent("scoreboard:getPlayers")
+AddEventHandler("scoreboard:getPlayers", function()
+  TriggerClientEvent("scoreboard:receivePlayers", source, players)
```

Comment:
Not latent event, also not rate limited. Good way for people to perhaps break net thread :/

Reply:
I will change this to a latent event, what would you recommend BPS wise for an event of this size?